easybuild/easyblocks/m/metis.py

```diff
@@ -125,7 +125,7 @@ def sanity_check_step(self):
 
         custom_paths = {
                         'files': ['bin/%s' % x for x in binfiles] + ['include/%s' % x for x in incfiles] +
-                                 ['lib/libmetis.a'],
+                                 [('lib/libmetis.a','lib/libmetis.so')],
```

Comment:
None of the current `METIS` installations include `libmetis.so`... So, this requires a particular configure option to be used?

If so, either include that in the easyblock, or don't include `libmetis.so` in the sanity check in the easyblock (you can easily override `sanity_check_paths` in an easyconfig that should produce it).